src/c-writer.cc

```diff
@@ -453,6 +450,8 @@ class CWriter {
 
   void PushFuncSection(std::string_view include_condition = "");
 
+  bool IsImport(const std::string& name) const;
```

Comment:
A good chunk of this PR seems to be adding and using this helper?  Can we maybe land this on its own to make this PR smaller?

Reply:
Sure, stand by...

Reply:
Split out to #2275